DragonKitchen/DragonKitchen/Views/TitleView.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  TitleView.swift
+//  DragonKitchen
+//
+//  Created by 김기영 on 2023/05/10.
+//
+
+import SwiftUI
+import NavigationStack
+
+struct TitleView: View {
+    var body: some View {
+        NavigationStackView(transitionType: .none) {
+            ZStack{
+                Image("TitleBackground")
+                    .resizable()
+                    .frame(width: UIScreen.width, height: UIScreen.height * 1.1)
```

Comment:
frame 설정을 이용해서 이미지 크기를 조절해주셨는데, 백그라운드 이미지가 위아래로 조금씩 잘려서 보이는 것 같아요. 대신에 Safe Area를 무시하는 설정을 해보시는건 어떨까요?

```suggestion
                    .ignoresSafeArea()
```